WooCommerce/Classes/ViewRelated/Editor/AztecEditorViewController.swift

```diff
@@ -46,6 +46,27 @@ final class AztecEditorViewController: UIViewController, Editor {
         return editorView.htmlTextView
     }
 
+    /// Aztec's Format Bar (toolbar above the keyboard)
+    ///
+    private lazy var formatBar: Aztec.FormatBar = {
+        let toolbar = AztecFormatBarFactory().formatBar() { [weak self] (formatBarItem, formatBar) in
+            guard let self = self else {
+                return
+            }
+            self.formatBarCommandCoordinator.handleAction(formatBarItem: formatBarItem,
+                                                          editorView: self.editorView,
+                                                          formatBar: formatBar)
+            formatBar.update(editorView: self.editorView)
+        }
+        return toolbar
+    }()
+
+    /// Aztec's Format Bar Action Handling Coordinator
+    ///
+    private lazy var formatBarCommandCoordinator: AztecFormatBarCommandCoordinator = {
```

Comment:
Both the format bar and the formatter coordinator are instantiated by the same class. I am just wondering, would we be better off if we just retained the factory instead of two instanced created by it?